cmd/dep/status.go

```diff
@@ -369,12 +369,10 @@ func runStatusAll(out outputter, p *dep.Project, sm *gps.SourceMgr) error {
 	rm, _ := ptree.ToReachMap(true, true, false, nil)
 
 	external := rm.Flatten(false)
-	roots := make(map[gps.ProjectRoot][]string)
-	var errs []string
+	roots := make(map[gps.ProjectRoot][]string, len(external))
 	for _, e := range external {
 		root, err := sm.DeduceProjectRoot(e)
 		if err != nil {
-			errs = append(errs, string(root))
 			continue
 		}
```

Comment:
`staticcheck` found this `errs` slice which is never used. At first I was going to return an `error` if the length of `errs` was greater than 0. Then I looked at the `DeduceProjectRoot` code and realized there are no cases where `err != nil` where `root` is not an empty string. So in the current code, if `DeduceProjectRoot` returns an `error`, `errs` will be a slice of empty strings.

I "fixed" the issue by deleting `errs` entirely. Should we do more to handle the error here?